satnogsclient/scheduler/tasks.py

```diff
@@ -164,24 +170,31 @@ def get_jobs():
 
 def task_feeder(port1, port2):
     sleep(1)
-    logger.info('Started task feeder')
-    print port1, ' ', port2
-    sock = Commsocket('127.0.0.1', port1)
-    sock.bind()
-    q = Queue(maxsize=1)
-    p = Process(target=task_listener, args=(port2, q))
-    p.daemon = True
-    p.start()
-    sock.listen()
-    while 1:
-        conn = sock.accept()
-        if conn:
-            conn.recv(sock.tasks_buffer_size)
-            if not q.empty():
-                conn.send(q.get())
-            else:
-                conn.send('[]')
-    p.join()
+    child_pid = 0
+    try:
+        logger.info('Started task feeder')
+        print port1, ' ', port2
```

Comment:
Please be consistent between using ``logging`` and ``print``. You should use python logging infrastructure everywhere.